libs/parsers/src/lib/models/operation.ts

```diff
@@ -99,7 +99,7 @@ interface DynamicTreePickerInfo {
   parameters?: Record<
     string,
     {
-      selectedItemValuePath: string;
```

Comment:
Do not remove this, this is set is manifest based operation so this is the new property connector will be using, just add the value-property one.